invenio_accounts/models.py

```diff
@@ -64,14 +65,17 @@ class Role(db.Model, Timestamp, RoleMixin):
 
     __tablename__ = "accounts_role"
 
-    id = db.Column(db.Integer(), primary_key=True)
+    id = db.Column(db.String(80), primary_key=True, default=str(uuid.uuid4()))
```

Comment:
Can use `UUIDType`, see for example here https://github.com/inveniosoftware/invenio-records/blob/master/invenio_records/models.py#L62